pkg/eosclient/eosbinary/eosbinary.go

```diff
@@ -703,6 +703,17 @@ func (c *Client) Rename(ctx context.Context, auth eosclient.Authorization, oldPa
 	return err
 }
 
+func (c *Client) ListWithRegex(ctx context.Context, auth eosclient.Authorization, path string, depth uint, regex string) ([]*eosclient.FileInfo, error) {
```

Comment:
ListFilesWithRegex?

Reply:
I can rename it if you want, chose this name because it is basically the same as `List`, just with a regex parameter added